pkg/autopilot/controller/setup.go

```diff
@@ -73,23 +75,28 @@ func (sc *setupController) Run(ctx context.Context) error {
 		}
 	}
 
-	hostname, err := apcomm.FindEffectiveHostname()
```

Comment:
It would probably be easier to use the same name for the ControlNode object that's used for the Kubernetes Node object. Maybe that's something we can achieve in config-v2.

Reply:
We have `AUTOPILOT_HOSTNAME` env var to override the ControlNode name.